Source/Scene/Billboard.js

```diff
@@ -98,6 +104,8 @@ define([
         this._scaleByDistance = options.scaleByDistance;
         this._translucencyByDistance = options.translucencyByDistance;
         this._pixelOffsetScaleByDistance = options.pixelOffsetScaleByDistance;
+        this._heightReference = defaultValue(options.heightReference, HeightReference.NONE);
+        this._maxImageSize = new Cartesian2(); // used by labels
```

Comment:
Probably `_maxSize` to keep it consistent, or...since `max` is not clear from context, perhaps get better symmetry with
* `_pickPrimitive` to `_ownerPrimitive`
* `_maxSize` to `_ownerSize` (and similar throughout)